service/src/main/java/com/commercetools/pspadapter/payone/IntegrationService.java

```diff
@@ -182,29 +178,15 @@ public int port() {
         final ImmutableMap<String, String> applicationInfo = ImmutableMap.of(
                 "version", serviceConfig.getApplicationVersion(),
                 "title", serviceConfig.getApplicationName());
-        Map<String, CompletionStage<Integer>> tenantMap = checkTenantStatuses(tenants);
-        //resolve all completable stages
-        listOfFuturesToFutureOfList(new ArrayList<>(tenantMap.values())).join();
-        //unpack completable features
-        Map<String, Integer> statusMap = tenantMap.keySet().stream()
-                .collect(toMap(tenant -> tenant, tenant -> tenantMap.get(tenant).toCompletableFuture().join()));
+
+        final Map<String, Integer> tenantStatutes = new HashMap<>();
```

Comment:
Why do we need these statuses? We are not checking anything tenant related, right?

Reply:
> We are not checking anything tenant related, right?
Yes.

> Why do we need these statuses? 
I also don't know why we needed those checks, I guess it's a multitenant based implementation, so I did not want to change the current response logic of health check. 
But we could, wdyt ? should we change it?

Reply:
We needed it because we were checking the CTP connection for each tenant's project. Since we're not checking CTP connection anymore, I'd say we don't need these statuses.

Reply:
Makes sense 👍

Reply:
f558a77